src/qt/pivx/send.cpp

```diff
@@ -727,7 +727,8 @@ void SendWidget::onContactsClicked(SendMultiRow* entry)
         menuContacts->setWalletModel(walletModel, AddressTableModel::Send);
         connect(menuContacts, &ContactsDropdown::contactSelected, [this](QString address, QString label) {
             if (focusedEntry) {
-                focusedEntry->setLabel(label);
+                if (!label.isNull() && label != "(no label)")
```

Comment:
i was seeing null labels in my early testing of this fix, but not after re-testing with a fresh wallet. I've left the check for the string literal `(no label)` because there really isn't a point in populating the label lineedit with that.

The string literal used here is also used when adding addresses to the address book when the user has not provided a label of their own

Reply:
Looking at the current code, the problem would arise when the label field is already complete. 
When a contact is selected, the address will be replaced but the label will remain being the same, which could cause confusion as the two addresses would appear to have the same label.

If we are not going to populate the `(no label)` string, should clear the label edit line.